addons/src/createNumberMask.js

```diff
@@ -27,6 +27,14 @@ export default function createNumberMask({
   const thousandsSeparatorSymbolLength = thousandsSeparatorSymbol && thousandsSeparatorSymbol.length || 0
 
   function numberMask(rawValue = emptyString) {
+    if(rawValue === '.' && allowDecimal) {
+      let mask = ['0', '.', /\d/]
+      if(prefix) {
+        mask.unshift(prefix)
```

Comment:
I wonder if the code below is better (or if it even makes a difference).. it would at least make it more consistent with how the prefix is currently already.

```js
if (prefixLength > 0) {
  mask = prefix.split(emptyString).concat(mask)
}
```

It doesn't seem to make any difference, and as I'm not the author of the original code here I don't know the reason/motivation for splitting the prefix before adding it in the first place... so maybe it doesn't make any difference.